apps/zero-api/src/users/entities/user.entity.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Exclude } from 'class-transformer';
+
+import { User, Role } from '@prisma/client';
+import { IsEnum } from 'class-validator';
+
+export class UserEntity implements User {
+  @ApiProperty({ example: 1 })
+  id: number;
+
+  @ApiProperty({ example: 'testuser1@foo.bar' })
+  email: string;
+
+  @ApiProperty({ example: 'John' })
+  firstName: string;
+
+  @ApiProperty({ example: 'Smith' })
+  lastName: string;
+
+  @ApiProperty({ enum: Role, enumName: 'Role'})
+  @IsEnum(Role)
+  role: Role
```

Comment:
Could You please rename field to "userRole"?

Reply:
sure